### PR TITLE
[FIX] pos_cash_rounding: equalize cash rounding

### PR DESCRIPTION
**Pull request by yelizariev**
TLDR: float_round in backend works differently with POS roundings in case of
negative amounts (refunds)

STEPS:

* setup rounding for POS, e.g. 0.1 DOWN
* make refund via POS for an amount with cents, e.g. -129.33
* proceed to POS payment
  * BEFORE: is shows -129.40
  * AFTER: is shows -129.30

WHY:

* This follows rule ``round(-x) = -round(x)`` -- this way we don't have
difference in absolute values for order and refund.
* See also https://github.com/odoo/odoo/commit/ea2c80cad018da6390ea9d116733de691907e1e4

---

opw-2357917

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by nim-odoo — APPROVED** [review]
Looks good to me.

@pimodoo ok for you?
robodoo delegate+

**Comment by pimodoo** [comment]
yes perfect for me

**Comment by yelizariev** [comment]
robodoo r+ пошустрее только - может еще успеем в 13 влить
